resourses/style.css

```diff
@@ -95,15 +95,15 @@ nav{
 /* HEADER */
 /* -------------------------------- */
 
-header{
+.banner{
   position: relative;
   height: 90vh;
   background-image: url("images/image1.jpg");
```

Comment:
Add CSS short hand property.

Reply:
Replaced background property with CSS short hands. But 'background-size' property couldn't be combined with rest of the background properties